test/k8sT/Policies.go

```diff
@@ -0,0 +1,274 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sTest
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/test/helpers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+)
+
+var _ = Describe("K8sPolicyTest", func() {
+
+	var demoPath string
+	var initialized bool
+	var kubectl *helpers.Kubectl
+	var l3Policy, l7Policy string
+	var logger *log.Entry
+	var path string
+	var podFilter string
+
+	initialize := func() {
+		if initialized == true {
+			return
```

Comment:
Did you consider doing this basic initialization using BeforeSuite()?

Reply:
Scratch that, I see that this is already used elsewhere in higher level initialization. I assume that precludes its usage within individual test files.